examples/media/index.html

```diff
@@ -216,13 +216,13 @@ <h2 class="container-filters-title">
       <a href="#" title="Back to top">
         <svg viewBow="0 0 40 40">
           <defs>
-            <linearGradient id="a" x1="100%" x2="0%" y1="100%" y2="0%">
+            <linearGradient id="d" x1="100%" x2="0%" y1="100%" y2="0%">
```

Comment:
https://github.com/algolia/instantsearch.js/blob/8f9a52feb0691f7511e84e3fa18cdc804538e408/examples/media/index.html#L58-L61

There is another `id="a"`. I think it collided with this. So I had to rename the id to display the background. I thought it was scoped somehow even though it's an `id`, but apparently, it's not..?
It's weird that this duplicated `id`s, it still works fine on Safari.

Reply:
Ah, good catch! We probably should have renamed them more explicitly.